CMakeLists.txt

```diff
@@ -1,23 +1,7 @@
 ### ---[ PCL global CMake
-cmake_minimum_required(VERSION 2.8 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.1 FATAL_ERROR)
```

Comment:
required for `CMAKE_CXX_STANDARD` which allows us to set CXX Standard in a portable manner. As a sideeffect allows removing Policies, which are a default in this CMake version

Reply:
Hey. We can now start merging C++14 stuff :)

Regarding the policies, it's ok to get rid of the ones which explicitly set the behavior to `NEW` but the ones setting to behavior to`OLD` need to be kept. We will address them on a separate PR, since they will require considerable changes.